bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Device.java

```diff
@@ -645,7 +645,8 @@ public Color getSystemColor (int id) {
 		case SWT.COLOR_DARK_GREEN:			pixel = 0x00008000;  break;
 		case SWT.COLOR_YELLOW:				pixel = 0x0000FFFF;  break;
 		case SWT.COLOR_DARK_YELLOW:			pixel = 0x00008080;  break;
-		case SWT.COLOR_BLUE:				pixel = 0x00FF0000;  break;
+		case SWT.COLOR_LIGHT_BLUE:			pixel = 0x00FF9933;  break;
```

Comment:
Adding a new system color for require adaptations of every OS implementation (i.e., also in the `Device` classes of MacOS and Linux). I am not sure whether we should really add a color for this use case. In particular, the color does not seem to have a specific meaning. In my Windows installation, the color currently used for the menu bar in dark mode does not correspond to this value, i.e., it may also depend on the OS version. Since it does not fit to the hightlight color of menu items either (see screenshot below) and since in light theme there is no similar hightlight color used as well (also see screenshot below).

![image](https://github.com/user-attachments/assets/31e53e1a-fd12-4c15-9f0b-6f674f08cc85)
![image](https://github.com/user-attachments/assets/a46ca1df-5f7a-4aed-beaa-c9d8cfcccf98)

I wonder whether we should not better just use one of the already defined color that we consider fitting. Since the hovering is also confirmed by opening the menu, a good visible color is not as important as, e.g., for menu items, where you want to see over which you currently hover. What about, for example, just using `COLOR_DARK_GRAY`? That would look as follows.
![image](https://github.com/user-attachments/assets/4d46f3c3-a957-4e1a-8375-8b0d0d2a0807)